docs/arch/adr-04-merging-mainstream-to-specialist-topics.md

```diff
@@ -0,0 +1,27 @@
+# Merging of Mainstream Topics into Specialist Topics
+
+## Status
+
+Proposed
+
+## Context
+
+GOV.UK currently maintains 2 separate topic trees that represent a navigational structure for users. Mainstream Browse is the oldest and considered legacy. Specialist Topics are newer but need tidying up and do not power breadcrumbs.
+
+Previous attempts to merge the two stalled (since 2015).
+
+The newest tree, Taxonomy, does not affect navigation and is treated as a purely metadata type categorisation tree and is ignored by this work.
+
+## Decision
+
+We initially planned to map each Mainstream Browse Topic to an equivalent within the Specialist Topics tree. This would have been complex and difficult to even begin the merging of the two trees. 
+
+But deeper discussion brought us to realise that there are no overlaps, allowing us to naively merge the trees and tidy up the data in one place. 
+
+## Consequences
+
+Once we've ported the topics across, we would have to periodically (interval TBD) sync data across from Mainstream Browse to Specialist Topics. 
```

Comment:
Is `sync data across from Mainstream Browse` correct lingo? Are you referring to the task of creating a new topic in the event of a publisher creating a new MSB page? It sort of sounds like we're saying that there will be a task to sync any pages (tagged to MSB) that are published or updated so that they are also tagged to the new specialist topic. But that's not the case. ie add some of this stuff from Jon:

```Proposal
Backend:
Follow original plan but keep all creation logic in Collections Publisher
Create and run rake task to create new topics based on existing browse pages, any new pages created from this are hidden (schema change).
Have logic to do this whenever a new Browse page is added in collections publisher
Modify patch links (tagging) behaviour to patch a relevant Topic whenever a Browse page is tagged in the two apps that allow it (Tagger and Publisher).
Should we have a one-off rake task to do this retroactively for existing content?
```

Reply:
Added.